scripts/templates/index.tmpl

```diff
@@ -23,7 +23,13 @@
     </div>
 
     <div id="content">
+        <h2>About SpatialReference.org</h2>
+        SpatialReference.org is a comprehensive online resource dedicated to providing detailed information and reference materials on spatial coordinate systems, projections, and transformations.<br>
```

Comment:
We do not provide any information about transformations (maybe some time in the future, but not near). About projections... well, only the information in the projected CRSs themselves.
```suggestion
        SpatialReference.org is a comprehensive online resource dedicated to providing detailed information and reference materials on spatial coordinate reference systems.<br>
```